src/osp/Active/physics.h

```diff
@@ -31,26 +31,57 @@ namespace osp::active
 {
 
 /**
- * Generic Rigid Body physics data
- *
- * TODO: possibly split into more components when the time comes
+ * @brief The ACtxPhysics struct
  */
-struct ACompRigidBody
+struct ACtxPhysics
 {
-    // Modify these
-    Vector3 m_inertia{1, 1, 1};
-    Vector3 m_netForce{0, 0, 0};
-    Vector3 m_netTorque{0, 0, 0};
+    Vector3 m_originTranslate;
```

Comment:
no default value?

Reply:
Vector3 default constructor initializes to (0, 0, 0)